Counterfeit Monkey Materials/Extensions/Dannii Willis/Ultra Undo.i7x

```diff
@@ -125,10 +127,11 @@ Global ultra_undo_needed = 0;
 	{
 		print (char) UUID_ARRAY->ix;
 	}
-	print "-", ultra_undo_counter;
+	print "-", (( ultra_undo_counter - 1 )  % ULTRA_UNDO_MAX_COUNT ) + 1;
+	! Translate ultra_undo_counter into a number between 1 and ULTRA_UNDO_MAX_COUNT
```

Comment:
Can you move this comment above the line please? And maybe phrase it something like "Take the mod of ultra_undo_counter ultra_undo_counter to keep the number of files to ULTRA_UNDO_MAX_COUNT"